e2e/hmr/tests/tests.spec.js

```diff
@@ -33,7 +34,8 @@ test.describe('pages', () => {
 		await expect(page.getByText('Index')).toBeVisible();
 
 		createFile('pages/new-page.md', 'This is a new page');
-		await waitForHMR(page);
+		// file deletions trigger full reload
+		// await waitForHMR(page);
```

Comment:
Remove the commented out code and improve the comment to explain

```suggestion
		// file deletions trigger full reload, so we don't waitForHMR() here
```